### PR TITLE
increase timeout

### PR DESCRIPTION
**Pull request by erezh31**
fixes: https://github.com/demisto/etc/issues/10650

### CODE REVIEW

**Comment by meirwah** [comment]
@anara123 @erezh31 can we merge this ?

File: Scripts/script-CBLiveGetFile.yml [hunk 1]
```diff
@@ -147,4 +147,5 @@ dependson:
   - cb-command-create
   - cb-command-info
   - cb-file-get
-timeout: 0s
+timeout: 300ns
```
Comment (anara123) [inline Scripts/script-CBLiveGetFile.yml]: ns=nanoseconds ?

**Comment by anara123** [comment]
@meirwah I run the build 3 times, if the these tests will not fail then we will merge

**Comment by erezh31** [comment]
@anara123 not sure it's smart to do it parallel... the query go to the same carbon black env but let's see

File: Scripts/script-CBLiveGetFile.yml (hunk 1 above)
Comment (erezh31) [reply Scripts/script-CBLiveGetFile.yml]: not sure... was wondering too. but this is the value when you change the automation timeout and download the yml.... suppose to in seconds

**Comment by anara123** [comment]
it seems your tests passed in all 3 builds

**Pull request merged**